packages/connectivity/src/scp-cf/environment-accessor/xsuaa.ts

```diff
@@ -44,19 +17,15 @@ export function clearXsuaaServices(): void {
 
 /**
  * @internal
- * @param options - Options on how to configure the XSUAA service.
  * @param options.disableCache - Value to enable or disable JWKS cache in xssec library. Defaults to false.
- * @param options.jwt - Either a JWT payload or an encoded JWT. Will be ignored if `credentials` are provided. If not provided, the first XSUAA service binding is used.
- * @param options.credentials - Xsuaa service credentials. If not provided, the credentials are fetched based on the JWT
- * @returns An instance of the xsuaa service that the application is bound to.
+ * @param options.credentials - Xsuaa service credentials. Required to create the xssec XSUAA instance.
+ * @returns An instance of {@code @sap/xssec/XsuaaService} for the provided credentials.
  */
-export function getXsuaaService(options?: {
+export function getXsuaaInstanceFromServiceCredentials(options: {
   disableCache?: boolean;
-  jwt?: JwtPayload | string;
-  credentials?: ServiceCredentials;
+  credentials: ServiceCredentials;
 }): any {
-  const credentials =
-    options?.credentials || getXsuaaServiceCredentials(options?.jwt);
+  const { credentials } = options;
```

Comment:
[pp] Can we make each property directly a parameter instead of having a mandatory `options` object?